### ISSUE TITLE
Feature request: can you add formula of LSTM to docs?

### ISSUE THREAD

**Issue by Mabinogiysk** [tensorflow/tensorflow#20435]
Have I written custom code： No
OS Platform and Distribution: Linux 16.04
TensorFlow installed from: pip install tensorflow-gpu
TensorFlow version: 1.8.0
CUDA/cuDNN version: 9.0

### Describe the problem
Hi,
I have trained a bidirectional lstm(256 hidden units) model in pytorch and I want to transfer it to tensorflow. 
In pytorch,  each LSTM has two weights and two bias:
```
rnn.0.rnn.weight_ih_l0 torch.Size([1024, 512])
rnn.0.rnn.weight_hh_l0 torch.Size([1024, 256])
rnn.0.rnn.bias_ih_l0 torch.Size([1024])
rnn.0.rnn.bias_hh_l0 torch.Size([1024])

weight_ih_l[k] – the learnable input-hidden weights of the kth layer (W_ii|W_if|W_ig|W_io), of shape (4*hidden_size x input_size)
weight_hh_l[k] – the learnable hidden-hidden weights of the kth layer (W_hi|W_hf|W_hg|W_ho), of shape (4*hidden_size x hidden_size)
bias_ih_l[k] – the learnable input-hidden bias of the kth layer (b_ii|b_if|b_ig|b_io), of shape (4*hidden_size)
bias_hh_l[k] – the learnable hidden-hidden bias of the kth layer (b_hi|b_hf|b_hg|b_ho), of shape (4*hidden_size)
```
However in tensorflow, each LSTM has only one weight and one bias:
```
<tf.Variable 'RNN/BiLSTM1/stack_bidirectional_rnn/cell_0/bidirectional_rnn/fw/basic_lstm_cell/kernel:0' shape=(768, 1024) dtype=float32_ref>,
 <tf.Variable 'RNN/BiLSTM1/stack_bidirectional_rnn/cell_0/bidirectional_rnn/fw/basic_lstm_cell/bias:0' shape=(1024,) dtype=float32_ref>,
```
I do not know how to convert the parameters in pytorch into tensorflow. Can you add some formula and weight structures of LSTM to docs as what they did in pytorch?
Thanks!

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by michaelisard** [comment]
Sorry for the delay in the reply. @MarkDaoust may be able to help with documentation. In the meantime you may get a better answer to the immediate question on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since there is also a larger community that reads questions there.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 50 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 65 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 80 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 95 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 110 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
